charts/sysdig-stackdriver-bridge/README.md

```diff
@@ -0,0 +1,94 @@
+
+## Introduction
+
+This chart adds the Sysdig Stackdriver Bridge application into a cluster using a simple `Deployment` with a `ConfigMap`.
+
+## Prerequisites
+
+- Sysdig should be deployed in the target Kubernetes cluster.
+- You will need to create a secret that matches the value `secret.name` (`stackdriver-webhook-bridge` by default) that contains the key `key.json` with the value of the created GCP IAM Service Account for Sysdig to access Stackbridge.
+- It is recommended to create this `Secret` in an out-of-band fashion in CI/CD, etc.
+  - Optionally, you can enable creation of the `Secret` via the chart by setting `secret.create=true` and providing the JSON data directly via `secret.data`. **This is not an advisable approach for production systems.**
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, run:
+
+```bash
+$ helm repo add sysdiglabs https://sysdiglabs.github.io/charts/
+```
+
+to add the `sysdiglabs` Helm chart repository. Then run:
+
+```bash
+$ helm install --name my-release sysdiglabs/sysdig-stackdriver-bridge
+```
+
+When done, the application should be running within your cluster if properly configured.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+> **Tip**: Use helm delete --purge my-release to completely remove the release from Helm internal storage
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Sysdig Stackdriver Bridge chart and their default values.
+
+| Parameter               | Description                                                                                                                                            | Default                                                             |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------- |
+| `image.repository`      | Image to use with Stackdriver bridge application.                                                                                                      | `sysdiglabs/stackdriver-webhook-bridge`                             |
+| `image.pullPolicy`      | Image pull policy.                                                                                                                                     | `IfNotPresent`                                                      |
+| `image.tag`             | Image tag.                                                                                                                                             | `latest`                                                            |
```

Comment:
We use fixed tags for this, it can be empty and it will honor the appVersion from Chart.yaml file